build-system/babel-config/import-resolver.js

```diff
@@ -41,14 +41,23 @@ function readJsconfigPaths() {
   return tsConfigPaths;
 }
 
+// Necessary for external modules that rely on React.
+const moduleAliases = {
+  'react': './src/react',
```

Comment:
Ah, this is great ... the previous attempt was much more complicated, but this is so simple, by redirecting `react -> .../react.js -> #preact` we ensure that we always get the build config from the `#preact` alias.  Nice!